back/src/Guide/Cli.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell   #-}
+
+-- | This module adds command line interface to run server.
+--
+-- See help with 'guide --help' to see available commands.
+module Guide.Cli
+       ( Command (..)
+       , parseCli
+       ) where
+
+
+import Imports
+
+import Data.Version (showVersion)
+import Development.GitRev (gitCommitDate, gitDirty, gitHash)
+import Options.Applicative (Parser, ParserInfo, command, execParser, fullDesc, help, helper, info,
```

Comment:
It would be better to use a qualified import for this one (`import qualified Options.Applicative as Opt`, for instance). Otherwise this list will have to be amended all the time as additional functions become used and some become unused.